ext/Rakefile

```diff
@@ -1,40 +1,48 @@
-# frozen_string_literal: true
```

Comment:
was there any problem with the literal here?

Reply:
nope, can add back in!